sw/device/silicon_creator/lib/drivers/keymgr.c

```diff
@@ -86,7 +86,7 @@ rom_error_t keymgr_state_advance_to_creator(const uint32_t binding_value[8],
   for (size_t i = 0; i < 8; ++i) {
```

Comment:
Should we use `ARRAYSIZE(binding_value->data)` ?

Reply:
Yes, done.